src/web/src/components/Posts/GitHubMobile.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  createStyles,
+  makeStyles,
+  Theme,
+  ListSubheader,
+  Accordion,
+  Typography,
+  AccordionSummary,
+  AccordionDetails,
+} from '@material-ui/core';
+import Repos from './Repos';
+import Issues from './Issues';
+import PullRequests from './PullRequests';
+import Commits from './Commits';
+import Users from './Users';
+import { VscGithub } from 'react-icons/vsc';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '0',
+      display: 'flex',
+      borderLeft: '1.5px solid #999999',
+      flexDirection: 'column',
+      [theme.breakpoints.up('lg')]: {
+        width: '21rem',
+      },
+      color: theme.palette.text.secondary,
+    },
+    GitHubInfoContainer: {
+      margin: '2rem 0 0 1rem',
+    },
+    icon: {
+      fontSize: '4rem',
+      marginRight: '1rem',
+      verticalAlign: 'text-bottom',
+      paddingLeft: '6rem',
+      alignItems: 'right',
+    },
+    accordionSummary: {
+      width: '10 rem',
+      paddingRight: '0',
+      justifyContent: 'flex-end',
+    },
+    accordion: {
+      backgroundColor: 'rgba(255, 255, 255, 0)',
```

Comment:
You can have it inherit main bg color: `backgroundColor: inherit`.

Reply:
Thanks for the recommendation, I learned a new thing. This is fixed, thanks!